data/css/alert.css

```diff
@@ -19,16 +19,16 @@ body .qwant-alert .qwant-alert__content {
 	border: 1px solid rgb(224, 226, 228);
 	border-radius: 2px;
 	margin: 8px auto;
-	padding: 0 8px;
+	padding: 0 4px;
 	line-height: 46px;
 	box-shadow: 0 0 8px rgba(0, 0, 0, 0.11);
 	background: white;
 }
 
-body .qwant-alert .qwant-alert__content--question {
+/*body .qwant-alert .qwant-alert__content--question {
```

Comment:
Yes

Louis AUDIBERT
Etudiant {EPITECH.} Nice
Promo 2018
louis.audibert@epitech.eu<mailto:louis.audibert@epitech.eu>
+33(0)6.27.99.13.77<tel:+33(0)6.27.99.13.77>

On 3 ao?t 2016, at 19:14, St?phane Erard <notifications@github.com<mailto:notifications@github.com>> wrote:


In data/css/alert.css<https://github.com/Qwant/firefox-extension/pull/5#discussion_r73366373>:

>       line-height: 46px;
>       box-shadow: 0 0 8px rgba(0, 0, 0, 0.11);
>       background: white;
>  }
>
> -body .qwant-alert .qwant-alert__content--question {
> +/*body .qwant-alert .qwant-alert__content--question {


If commented : remove ?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Qwant/firefox-extension/pull/5/files/d481b38f7e39e63a6d5b9c44c1eae774a225a09f#r73366373>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AGfWcFuiRfSMkOM2iGGH2P8i_9XLfRymks5qcMlAgaJpZM4JbtK2>.